### DISCUSSION TITLE
Family Library Sharing functional

### DISCUSSION THREAD

**Discussion by ksardas2015** [chr233/ASFEnhance#17] [功能建议 / Ideas]
I would like, if possible, to teach ASF how to manage Family Library Sharing. On my main account, I allowed 5 bots to use my games library so they can play a game that was only purchased on my main account. And everything works fine, all accounts can play the game at the same time, but the problem is that for some reason, sometimes this checkmark becomes unchecked on my main account.
![Screenshot_1](https://user-images.githubusercontent.com/38585239/174293163-db4106b4-28c7-449c-8590-37e6cabe6282.png)
Because of this, access to the game library becomes closed. I would like ASF to be able to give access from one account library of to other accounts (to check this checkbox). You can add more features related to Family Library Sharing. For example control which accounts will be given access. But it's not that important, I'm more worried about this checkbox :))
In any case, these are just my ideas for additional functionality for the plugin. Thanks!

**Comment by chr233** [comment]
I think that could be very difficult to implement, I will try to figueout if it's possible to impl this function after the summer sale ended.

**Reply by ksardas2015** [reply]
is there any success?
